sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -251,7 +243,7 @@ protected AmqpConnectionScope()
             var connection = await ActiveConnection.GetOrCreateAsync(timeout).ConfigureAwait(false);
             cancellationToken.ThrowIfCancellationRequested<TaskCanceledException>();
 
-            var link = await CreateReceivingLinkAsync(
+            ReceivingAmqpLink link = await CreateReceivingLinkAsync(
```

Comment:
Is there a reason not to use `var`?

Reply:
We have a StyleCop rule to avoid using var except when used with constructors or literals where it is obvious what the type would be. This should make it easier to see the type without digging into the code that is being called.

Reply:
Got it, thanks.